ts/features/bonus/siciliaVola/screens/voucherList/VoucherListScreen.tsx

```diff
@@ -156,7 +195,10 @@ const VoucherListScreen = (props: Props): React.ReactElement => {
       <SafeAreaView style={IOStyles.flex} testID={"VoucherListScreen"}>
         {!isReady(props.possibleVoucherState) ? (
           <LoadingErrorComponent
-            isLoading={isLoading(props.possibleVoucherState)}
+            isLoading={
+              isLoading(props.possibleVoucherState) &&
+              !isError(props.uiParameters.requiredDataLoaded)
+            }
```

Comment:
With the introduction of the `LoadingErrorComponent` at line `160` which covers loading both remote data, this other one and the conditional rendering can be removed, right? 
The condition 
```typescript
  if (
    !isReady(props.possibleVoucherState) ||
    ((isDataLoadedError || isDataLoadedLoading) &&
      !isFirstPageLoadedSuccesfully)
  ) 
```
should already cover this case.

Reply:
Stupid error, sorry! I forgot to remove it. Yes, all the case are already covered with the condition you mentioned.
Removed in [98ed26e](https://github.com/pagopa/io-app/pull/3361/commits/98ed26edf69b55d8519a81b5bda0332ea64f29cc)